ReGuilded/inject/index.js

```diff
@@ -1,33 +1,39 @@
 const { exec } = require("child_process");
 const inject = require("./inject.js");
+const log4js = require('log4js')
+
+const logger = log4js.getLogger('ReGuilded')
+logger.level = 'debug'
 
 // TODO: Figure out what directory Guilded is in on derwin and what the command is to force close it.
 let platformModule;
 try {
     platformModule = require(`./platforms/${process.platform}.js`);
 } catch (err) {
-    if (err.code === "MODULE_NOT_FOUND") {
-        console.log(`[ReGuilded] | Your platform, ${process.platform}, does not seem to be supported yet.\n[ReGuilded] | Go ahead and create a new Issue on GitHub, make sure to include that you're attempting to use it on "${process.platform}"!\n[ReGuilded] | https://github.com/ReGuilded/ReGuilded/issues/new`);
-        return;
-    }
+    if (err.code === "MODULE_NOT_FOUND")
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/ReGuilded/ReGuilded/pullRequest?prid=7536470)
```suggestion
    if (err.code === "MODULE_NOT_FOUND")
```